### ISSUE TITLE
username and password cannot be blank

### ISSUE THREAD

**Issue by TechBill** [DanielnetoDotCom/YouPHPTube-Encoder#126]
I installed the encoder and when I tried to sign in. I get a Sorry your username and password cannot be blank.  I check the encoder DB and the username and password is listed in it. The password is not even encrypted. 

What would cause this message and not let me sign in?

Thank you

**Comment by DanielnetoDotCom** [comment]
Sorry, I really never seen this. please check your log file to try to find a clue

**Comment by TechBill** [comment]
Should it work if encoder was installed as sub-folder? like domain.com/encoder ?

I created a sub-domain encoder.domain.com then tried to install it but it get stuck at "Processing" on sub-domain installation - the DB never get created 
![screen shot 2019-03-07 at 3 28 42 pm](https://user-images.githubusercontent.com/945103/53990376-bf27e380-40ed-11e9-8f34-0822cc62bf04.jpg)

**Comment by TechBill** [comment]
This what I get when tried to log in on sub-folder domain.com/encoder/ 

![screen shot 2019-03-07 at 3 30 10 pm](https://user-images.githubusercontent.com/945103/53990487-02825200-40ee-11e9-9acf-0296ff8357ce.jpg)

**Comment by TechBill** [comment]
Found this error in log for sub-domain installation 

[07-Mar-2019 15:06:17 America/Chicago] PHP Warning:  require_once(/home/proscoops/encoder.proscoops.com/view/../videos/configuration.php): failed to open stream: No such file or directory in /home/proscoops/encoder.proscoops.com/view/index.php on line 18

**Comment by DanielnetoDotCom** [comment]
yes, this may be the problem

how is your encoder configuration.php file?

**Comment by TechBill** [comment]
Sorry I have been busy with other projects.  

Where should the configuration file be located in? I do see one in /objects/ folder however it look like the error log expect it in a different folder? 

/home/proscoops/encoder.proscoops.com/view/../videos/configuration.php

What folder is /../?

**Comment by DanielnetoDotCom** [comment]
should be inside your videos directory

**Comment by TechBill** [comment]
How do I change the php it using?  I noticed that it detect php 5.6 when the server have php 7.2 installed.   I wonder that is the problem?  Cpanel allows several version of php to be install but look like your script is pointed to /usr/bin/php?  Is there a way to change and point to a different php bin?

**Comment by TechBill** [comment]
Never mind, strange reason it detect php 7.2 now

**Comment by DanielnetoDotCom** [comment]
is it fixed now?

**Comment by TechBill** [comment]
I am still working on it.  I have not been able to get it to work yet.

**Comment by raffeal** [comment]
same error

**Comment by raffeal** [comment]
no problem in configuration file

**Comment by DanielnetoDotCom** [comment]
I can not reproduce this error, you may have something redirecting or blocking your request.

**Comment by TechBill** [comment]
I am still looking into this.   Have this been tested on php 7.2 before?

**Comment by DanielnetoDotCom** [comment]
@TechBill Yes, I use PHP 7.2 on any Ubuntu 18 installation with no problem

**Comment by TechBill** [comment]
Do we use same database and mysql user for streamer and encoder site?

**Comment by raffeal** [comment]
i have installed both in godaddy vps...and encoder is not login in 
and also live rtmp is not connected and stream key is hidden

**Comment by raffeal** [comment]
Do we use same database and mysql user for streamer and encoder site?

same question..... i have created two different database

**Comment by TechBill** [comment]
I was able to sign in today.  I have to use and set up ssl cerf to make it work

**Comment by Khaled4** [comment]
I faced all of the problems listed above. I was finally able to connect and login when I rolled back php to version 7.2
